app-connector-client/server/resources/api.yaml

```diff
@@ -1,20 +1,10 @@
 openapi: '3.0.0'
 info:
-  version: '0.1.0'
+  version: '1.3.20'

```

Comment:
Let's use something like 1, we will not update the version number every time we release a new package

Reply:
version is 1.0.0

Reply:
Can we use something like 1, without further digits